current-structure.md

```diff
@@ -15,9 +18,9 @@ Try to organise who will be running the next meetup on the Monday **before** so
 * Introduce people to gitter channel
 
 **6.15pm**
-* Give an initial presentation
+* Give an initial presentation **explaining about the course and encourage people to apply**
```

Comment:
Also, Twitter Content:
+ Find out who in the room is using twitter (*get their handles*)
+ Take photo of room when full and tag all the people who gave you handles and encourage them to re-tweet/star